app/routes/errors/HTTPError.tsx

```diff
@@ -14,8 +14,8 @@ const HTTPMapping = {
 };
 const fallbackStatus = 404;
 
-const getHTTPError = (statusCode) =>
-  HTTPMapping[statusCode] || HTTPMapping[fallbackStatus];
+const getHTTPError = (statusCode?: number) =>
+  HTTPMapping[statusCode || fallbackStatus] || HTTPMapping[fallbackStatus];
```

Comment:
```suggestion
const getHTTPError = (statusCode?: number) =>
  HTTPMapping[statusCode || fallbackStatus];
```
Is this not the same?

Reply:
Oh, hahah. Yeah it is:)